test/server/config_validation/server_test.cc

```diff
@@ -267,12 +267,11 @@ TEST_P(ValidationServerTest, WithProcessContext) {
 // as-is. (Note, /dev/stdout as an access log file is invalid on Windows, no equivalent /dev/
 // exists.)
 
-auto testing_values =
-    ::testing::Values("front-proxy_envoy.yaml", "envoyproxy_io_proxy.yaml",
+auto testing_values = ::testing::Values("front-proxy_envoy.yaml", "envoyproxy_io_proxy.yaml",
 #if defined(WIN32) && defined(SO_ORIGINAL_DST)
-                      "configs_original-dst-cluster_proxy_config.yaml",
+                                        "configs_original-dst-cluster_proxy_config.yaml",
 #endif
-                      "grpc-bridge_server_envoy-proxy.yaml", "front-proxy_service-envoy.yaml");
```

Comment:
`grpc-bridge_server_envoy-proxy.yaml` was using extensions inside and was failing. It is almost identical to `front-proxy_service-envoy.yaml` here, so it should be fine to just remove it.

Reply:
I'm a bit confsued by this comment, does this PR make previously valid configs now stop passing tests?

Reply:
not really. This config validation test case (grpc-bridge_server_envoy-proxy.yaml) required the extension compiled in this integration test build target, and that's why it failed if we remove the extension dependency here.

Having said that, I found the fix without requiring test (actually `//test/server:conifg_validation` is allowed to have extensions target in the visibility policy). I will push the commit soon.

Reply:
https://github.com/envoyproxy/envoy/pull/35049/commits/3a8eb40c08ce7ea0d5b0d2d3c6ca9d3bd4a6620b done!